docs/FeatureManagement/Samples/TargetingFilter.modified.tests.json

```diff
@@ -0,0 +1,90 @@
+[
+    {
+        "TestID": "TargetingFilterModified-RolloutPercentageUpdate-Aiden62",
+        "FeatureFlagName": "RolloutPercentageUpdate",
+        "Inputs": {"user":"Aiden"},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, 62% default rollout Aiden is part of it."
+    },
+    {
+        "TestID": "TargetingFilterModified-RolloutPercentageUpdate-AidenStage162",
```

Comment:
Better to use the "Aiden-Stage1-62" instead of "AidenStage162".

Reply:
I don't mind us changing these around a bit once we get everything merged in. I just would like this as a baseline so I can start expanding on the tests. We mainly have positive use cases here + Variants just got merged into python and I would like to add those too.

Reply:
>I don't mind us changing these around a bit once we get everything merged in. 
Do you mean we could merge this PR first, and then update the `TestID` property once the format is settled down?

Reply:
sure

Reply:
I don't entirely see the reason for IDs. These are already identifiable as {filename}[index]. I'd rather have no id field and just have a friendly name for the test. Or no name at all.

Reply:
@rossgrambo what index? They aren't in a list. This is the ID we print out when the test fails.

Reply:
The top level is an array. The index would be it's place in the array. ie: `json.parse("tests.json")[index]`.

The reason I say this is because the ID is essentially a composite key we could deduce rather than manually define/maintain. IE:

```python
id = fileName + "." + FeatureFlagName + "." + friendlyName
```

But as you said- you want them unique, so:

```python
id = fileName + "." + index + "." + FeatureFlagName + "." + friendlyName
```

And this id doesn't need to actually exist, but just be print out when a test fails.

Reply:
Alternatively we could this for a unique id:
```python
uniqueId = fileName + "." + index
```

But when a test fails only use the friendly name. Maybe including ID if the friendly name isn't clear enough.
```python
message = "Exception in " + friendlyName + "(" + uniqueId + ")"
```